store/tikv/oracle/oracle.go

```diff
@@ -33,6 +33,7 @@ type Oracle interface {
 	GetTimestampAsync(ctx context.Context, opt *Option) Future
 	GetLowResolutionTimestamp(ctx context.Context, opt *Option) (uint64, error)
 	GetLowResolutionTimestampAsync(ctx context.Context, opt *Option) Future
+	GetStaleTimestamp(ctx context.Context, prevSecond int64) (uint64, error)
```

Comment:
```suggestion
	GetStaleTimestamp(ctx context.Context, prevSecond uint64) (uint64, error)
```

Reply:
Thanks for your review, I agree with you.